Example/Podfile.lock

```diff
@@ -173,13 +173,13 @@ EXTERNAL SOURCES:
     :branch: fetch-user-details
     :git: https://github.com/artsy/Artsy-Authentication.git
   Emission:
-    :path: ../
+    :path: "../"
```

Comment:
Yeah… I’m on Ruby 2.3.0 in this project. I guess one thing we could do is consolidate on a Ruby version (and add a `.ruby-version` file) ?